### ISSUE TITLE
No module named 'continuedev.src'

### ISSUE THREAD

**Issue by thinkout-pm** [continuedev/continue#526]
### Before submitting your bug report

- [ ] I believe this is a bug. I'll try to join the [Continue Discord](https://discord.gg/NWtdYexhMs) for questions
- [ ] I'm not able to find an [open issue](https://github.com/continuedev/continue/issues?q=is%3Aopen+is%3Aissue) that reports the same bug
- [ ] I've seen the [troubleshooting guide](https://continue.dev/docs/troubleshooting) on the Continue Docs

### Relevant environment info

```Markdown
- OS:  windows 
- Continue:
```

### Description

python -m continuedev

### Log output

```Shell
Traceback (most recent call last):

  File "C:\Users\Danila\AppData\Local\Programs\Python\Python311\Lib\site-packages\continuedev\core\sdk.py", line 76, in create
    sdk.config = config or sdk._load_config_dot_py()
                           ^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\Users\Danila\AppData\Local\Programs\Python\Python311\Lib\site-packages\continuedev\core\sdk.py", line 243, in _load_config_dot_py
    config = ContinueConfig.from_filepath(path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "C:\Users\Danila\AppData\Local\Programs\Python\Python311\Lib\site-packages\continuedev\core\config.py", line 112, in from_filepath
    spec.loader.exec_module(config)

  File "<frozen importlib._bootstrap_external>", line 940, in exec_module

  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed

  File "C:\Users\Danila\.continue\config.py", line 7, in <module>
    from continuedev.src.continuedev.core.models import Models

ModuleNotFoundError: No module named 'continuedev.src'
```

**Comment by brytonsf** [comment]
I'm getting this as well, but on a Mac M1. I run `python -m continuedev`, the app starts up, and then I send a message/open a connection to the server, and get the same error:
`[2023-10-02 11:39:27,690] [ERROR] Failed to load config.py: ['Traceback (most recent call last):\n', '  File "/opt/homebrew/lib/python3.11/site-packages/continuedev/core/sdk.py", line 76, in create\n    sdk.config = config or sdk._load_config_dot_py()\n                           ^^^^^^^^^^^^^^^^^^^^^^^^^\n', '  File "/opt/homebrew/lib/python3.11/site-packages/continuedev/core/sdk.py", line 243, in _load_config_dot_py\n    config = ContinueConfig.from_filepath(path)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n', '  File "/opt/homebrew/lib/python3.11/site-packages/continuedev/core/config.py", line 112, in from_filepath\n    spec.loader.exec_module(config)\n', '  File "<frozen importlib._bootstrap_external>", line 940, in exec_module\n', '  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed\n', '  File "/Users/bryton/.continue/config.py", line 7, in <module>\n    from continuedev.src.continuedev.core.models import Models\n', "ModuleNotFoundError: No module named 'continuedev.src'\n"]`

It still lets me chat/use Continue in VSCode, though. My local model also isn't being recognized (I changed config.py) so I assume it's falling back to some sort of defaults.

**Comment by sestinj** [comment]
@brytonsf @thinkout-pm this is a bug I will be fixing in the next couple days, but for now this is due to difference in the import path when running continue manually. If you replace all `continuedev.src.continuedev.` with continuedev.` in config.py, you'll no longer see this error

I'll share an update here once this is resolved

**Comment by sestinj** [comment]
@brytonsf @thinkout-pm I've just released a new version (v0.0.409 on VS Code Marketplace, and 0.1.49 on PyPI) that will automatically make this replacement. Next up is to solve the root cause

Let me know if there are any other problems and I can re-open, but I'm going to close this issue since I'd like any new problems to be reported separately.
